mmv1/templates/terraform/constants/cloudfunctions2_function.go.erb

```diff
@@ -0,0 +1,14 @@
+// Suppress diffs for the system environment variables
+func environmentVariablesDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	if k == "service_config.0.environment_variables.LOG_EXECUTION_ID" && new == "" {
+		return true
+	}
+
+	// Let diff be determined by environment_variables (above)
+	if strings.HasPrefix(k, "service_config.0.environment_variables.%") {
```

Comment:
Does this ignore all diffs on environment_variables? Why do we need the above lines then?